.travis.yml

```diff
@@ -6,8 +6,7 @@ dist: trusty
 
 env:
   matrix:
-    - PYTHON=3.6 TESTS=true COVERAGE=true PACKAGES="lz4" TORNADO=5 CRICK=true
-    - PYTHON=3.7 TESTS=true PACKAGES="python-snappy python-blosc" TORNADO=6
+    - PYTHON=3.7 TESTS=true COVERAGE=true PACKAGES="lz4 python-snappy python-blosc" CRICK=true TORNADO=5
```

Comment:
I think here you removed the test case for when snappy and blosc are not installed

Reply:
I think the new Python 3.7 environment file is not including snappy or blosc, but please recheck and make sure this looks ok 🙂